components/LessonTabs/LessonTabs.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { CURRICULUM_PATH } from '../../constants'
+import NavCard from '../NavCard'
+
+type Lesson = {
+  title: string
+  docUrl?: string | null
+  slug: string
+}
+
+const LessonTabs = ({
+  lesson,
+  activeTab
+}: {
+  lesson: Lesson
+  activeTab: string
```

Comment:
should we restrict the activeTab type to only certain inputs like for example `'lesson' | 'exercises' | 'challenge' | 'mentor exercises' `?